src/styles/blocks/calendar.scss

```
@@ -0,0 +1,49 @@
+.calendar {
+  padding: 100px 0;
+  display: flex;
+  flex-wrap: wrap;
+  width: CalcMaxWidth($columns-number, $calendar-day-width, $calendar-gap);
+  gap: $calendar-gap;
+  
+  &--container{
+    margin: auto;
+  }
```

```suggestion
.calendar {
  padding: 100px 0;
  display: flex;
  flex-wrap: wrap;
  width: CalcMaxWidth($columns-number, $calendar-day-width, $calendar-gap);
  gap: $calendar-gap;
```

Find another way to center the calendar on the screen (tip - add styles to main).